Sources/MapboxNavigation/RoutesPresentationStyle.swift

```diff
@@ -8,14 +10,17 @@ public enum RoutesPresentationStyle {
     
     /**
      Only first route will be presented on a map view.
+     
+     - parameter cameraOptions: The custom `CameraOptions` that the map view ill be presented with.
+     If no value provided, the map view will present the whole first route from above.
      */
-    case single
+    case single(cameraOptions: CameraOptions? = nil)
     
     /**
      All routes will be presented on a map view.
      
-     - parameter shouldFit: If `true` geometry of all routes will be used for camera transition.
-     If `false` geometry of only first route will be used. Defaults to `true`.
+     - parameter cameraOptions: The custom `CameraOptions` that the map view ill be presented with.
+     If no value provided, the map view will present all routes from above.
      */
-    case all(shouldFit: Bool = true)
+    case all(cameraOptions: CameraOptions? = nil)
```

Comment:
I think that `shouldFit` property should stay, as it's used to create geometry to either fit all routes/or fit only first route. `CameraOptions` should stay, as it's going to be used by application level to define padding, bearing or pitch where this geometry will be shown.

Please use this API in application, it'll give you better understanding about its usage.

Reply:
I see. So the `shouldFit` is provided for the geometry information of the `CameraOptions`. So what if users allow the `shouldFit` be true and also provide the custom `cameraOptions`? Because the geometry of the routes/route would limit the zoom and center. Or in this case, should we only adopt the custom `padding, bearing, pitch` from the `cameraOptions` and the geometry from the routes/route?

Reply:
I think second one, we should use geometry of the route(s), but `CameraOptions` from the another property, if it's not nil. `shouldFit` only allows to make a decision how geometry will look, not `CameraOptions`.

Reply:
Got it. So right now the `geometry` is determined by the `single/all` and the `shouldFit`, while the `padding, bearing, pitch` is determined by the `CameraOptions`, with the default padding be the safe area.